### PR TITLE
[Refactor]: user schema

### PR DESCRIPTION
**Pull request by kalash-copods**
created is_curr_address_same field in user schema and modified create user and bulk user add views accordingly.

### CODE REVIEW

File: UserApp/migrations/0006_user_is_curr_address_same.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('UserApp', '0005_alter_user_work_type'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='user',
+            name='is_curr_address_same',
```
Comment (richa-copods) [inline UserApp/migrations/0006_user_is_curr_address_same.py:15]: give full name: is_current_address_same

File: UserApp/views/user_views.py [hunk 2]
```diff
@@ -95,6 +95,12 @@ def createUser(request):
     if request.method=='POST':
         try:
             user_data = JSONParser().parse(request)
+            if user_data.get('is_curr_address_same'):
```
Comment (richa-copods) [inline UserApp/views/user_views.py:98]: No need of doing this.

File: UserApp/views/user_views.py [hunk 3]
```diff
@@ -192,6 +198,11 @@ def bulkUserAdd(request):
             department_serializer = DepartmentSerializer(department)
             user['department'] = department_serializer.data['id']
 
+            if user.get('is_curr_address_same'):
+                user['current_address_line'] = user.get('permanent_address_line')
```
Comment (richa-copods) [inline UserApp/views/user_views.py:202]: No need of doing this

**Review by richa-copods — CHANGES_REQUESTED** [review]

File: UserApp/views/user_views.py (hunk 2 above)
Comment (kalash-copods) [reply UserApp/views/user_views.py:98]: done

File: UserApp/views/user_views.py (hunk 3 above)
Comment (kalash-copods) [reply UserApp/views/user_views.py]: done

File: UserApp/models.py [hunk 4]
```diff
@@ -81,6 +80,7 @@ class User(models.Model):
     permanent_address_city = models.CharField(max_length=100, null=True)
     permanent_address_state = models.CharField(max_length=100, null=True)
     permanent_address_pincode = models.IntegerField(null=True)
+    is_current_address_same = models.BooleanField(default=True)
```
Comment (richa-copods) [inline UserApp/models.py:83]: is_current_same_as_permanent_address

**Review by richa-copods — CHANGES_REQUESTED** [review]

**Pull request merged**